code/modules/mob/living/carbon/human/whisper.dm

```diff
@@ -14,6 +14,17 @@
 	else
 		speaking = get_default_language()
 
+	// Check if the speech requires dexterity, and if so, stop if can't use hands.
+	if(speaking && (speaking.flags & NEEDS_DEX))
+		if(src.incapacitated() || src.handcuffed)

```

Comment:
this is redundant. You don't need to check for handcuffing if you're checking for incap'd.